src/pyhf/infer/calculators.py

```diff
@@ -251,7 +268,7 @@ def teststatistic(self, poi_test):
         """
         tensorlib, _ = get_backend()
 
-        teststat_func = qmu_tilde if self.qtilde else qmu
+        teststat_func = test_statistics.get(self.test_stat)
```

Comment:
Only applies if people agree on the API comment

```suggestion
        teststat_func = test_statistics.get_test_stat(self.test_stat)
```